posthog/management/commands/revert_environment_beta.py

```diff
@@ -0,0 +1,140 @@
+            self.stdout.write("No organization IDs provided. Exiting.")
+            return
+
+        organization_ids = [int(org_id.strip()) for org_id in organization_ids_input.split(",") if org_id.strip()]
+
+        if not organization_ids:
+            self.stdout.write("No valid organization IDs provided. Exiting.")
+            return
+
+        self.stdout.write(
+            f"Processing {len(organization_ids)} organization(s): {', '.join(map(str, organization_ids))}"
+        )
+
+        for org_id in organization_ids:
+            self.stdout.write(f"\nProcessing organization with ID: {org_id}")
+            self.revert_environment_beta(org_id)
+
+        self.stdout.write("\nAll organizations processed successfully.")
+
+    def revert_environment_beta(self, organization_id: int):
+        """
+        Revert environment beta by ensuring each team belongs to exactly one project.
+
+        For each project with multiple teams, create new projects for teams where team.id != project.id,
+        and move those teams to the new projects.
+
+        Args:
+            organization_id: The ID of the organization to process
+        """
+        logger = logging.getLogger(__name__)
+        logger.info(f"Starting environment beta reversion for organization {organization_id}")
+
+        try:
+            # Get the organization
+            organization = Organization.objects.get(id=organization_id)
+            logger.info(f"Found organization: {organization.name}")
+
+            # Get all projects for this organization
+            projects = Project.objects.filter(organization_id=organization_id)
+            logger.info(f"Found {projects.count()} projects for organization {organization.name}")
+
+            # Process each project
+            for project in projects:
+                # Get all teams for this project
+                teams = Team.objects.filter(project_id=project.id)
+                team_count = teams.count()
+
+                if team_count <= 1:
+                    logger.info(f"Project {project.id} ({project.name}) has {team_count} team(s). No action needed.")
+                    continue
+
+                logger.info(f"Project {project.id} ({project.name}) has {team_count} teams. Processing...")
+
+                # Process each team in the project
+                for team in teams:
+                    # Skip the team if its ID matches the project ID (this team stays with the current project)
+                    if team.id == project.id:
+                        logger.info(f"Team {team.id} ({team.name}) matches project ID. Keeping in current project.")
+                        continue
+
+                    # Create a new project for this team
+                    new_project_id = team.id
+
+                    with transaction.atomic():
+                        existing_project = Project.objects.filter(id=new_project_id).first()
+                        if existing_project:
```

Comment:
Slightly more readable:
```suggestion
                        does_project_with_same_id_exist = Project.objects.filter(id=new_project_id).exists()
                        if does_project_with_same_id_exist:
```